adapters/kargo/kargotest/exemplary/banner.json

```diff
@@ -0,0 +1,207 @@
+{
+  "mockBidRequest": {
+    "id": "5f4d1e01",
+    "at": 1,
+    "imp": [
+      {
+        "id": "8b1bdcca",
+        "banner": {
+          "w": 300,
+          "h": 300,
+          "format": [
+            {
+              "w": 300,
+              "h": 250
+            }
+          ],
+          "pos": 0,
+          "expdir": [1, 2, 3, 4, 5]
+        },
+        "tagid": "73",
+        "secure": 1,
+        "iframebuster": ["ALL"],
+        "ext": {
+          "adSlotID": "11523"
+        }
+      }
+    ],
+    "site": {
+      "id": "123",
+      "domain": "www.dailymail.co.uk",
+      "cat": ["IAB7"],
+      "page": "https://www.dailymail.co.uk/",
+      "ref": "https://www.dailymail.co.uk/",
+      "mobile": 1,
+      "publisher": {
+        "id": "1"
+      }
+    },
+    "device": {
+      "ua": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.1 Safari/537",
+      "geo": {
+        "ipservice": 4,
+        "country": "US",
+        "region": "CO",
+        "metro": "751",
+        "city": "SomeCityInCo",
+        "zip": "11223"
+      },
+      "ip": "127.0.0.1",
+      "devicetype": 2,
+      "make": "Apple",
+      "model": "iPhone",
+      "os": "Windows",
+      "osv": "10.0",
+      "carrier": "none",
+      "language": "en",
+      "connectiontype": 2,
+      "dnt": 0
+    },
+    "user": {
+      "id": "07fb48ed",
+      "ext": {
+      },
+      "buyeruid": "345"
+    },
+    "regs": {
+      "ext": {
+        "us_privacy": "1"
+      }
+    },
+    "tmax": 200,
+    "ext": {
+    }
+  },
+  "httpCalls": [
+    {
+      "expectedRequest": {
+        "uri": "http://example.com/bid",
+        "body": {
+          "id": "5f4d1e01",
+          "at": 1,
+          "imp": [
+            {
+              "id": "8b1bdcca",
+              "banner": {
+                "w": 300,
+                "h": 300,
+                "format": [
+                  {
+                    "w": 300,
+                    "h": 250
+                  }
+                ],
+                "pos": 0,
+                "expdir": [1, 2, 3, 4, 5]
+              },
+              "tagid": "73",
+              "secure": 1,
+              "iframebuster": ["ALL"],
+              "ext": {
+                "adSlotID": "11523"
```

Comment:
Each impression will have an `Imp.Ext.adSlotID` and a `Imp.tagid`. Since the `adSlotID` is now required by prebid, Kraken can change to using this instead of appending `"siteID" + . + "tagId"` as shown here: https://github.com/KargoGlobal/kraken/blob/master/bidrequest/bidrequest.go#L605